websocket/config.go

```diff
@@ -185,6 +199,8 @@ func (t Topic) prefix() string {
 		return "AM"
 	case IndexValue:
 		return "V"
+	case BusinuessFairMarketValue:
```

Comment:
@aitzkovitz just noticed this is spelled wrong

```suggestion
	case BusinessFairMarketValue:
```

Reply:
fixed